src/libsystemd/sd-journal/journal-file.c

```diff
@@ -2205,7 +2205,9 @@ int journal_file_append_entry(
                 return r;
 #endif
 
-        items = newa(EntryItem, n_iovec);
+        items = new(EntryItem, n_iovec);
```

Comment:
Sorry, I forgot to leave this feedback earlier. This function is called very often when logging, so I'm not sure if we're OK with suddenly allocating here. Can we drop this commit for now? If we start allocating memory here I'd prefer to be smarter about it and to reuse the allocated buffer instead of allocating a new one every time.

Reply:
OK.